README.md

```diff
@@ -75,17 +75,18 @@ Russian translation is maintained by Translation Gang.
 Spanish translation is maintained by:
 
 [1950Labs](https://1950labs.com) & [Vue.js Montevideo](https://www.meetup.com/Montevideo-Vue-JS-Meetup/):
+- [Leonel More](https://github.com/leonelmore) | [Twitter](https://twitter.com/leonelmore)
 - [Sebastián Camacho](https://github.com/sxcamacho) | [Twitter](https://twitter.com/sxcamacho)
 - [Diana Rodríguez](https://github.com/alphacentauri82) | [Twitter](https://twitter.com/cotufa82)
-- [Alejandro](https://github.com/alejandro8605)
+- [Alejandro Parada](https://github.com/alejandro8605)
 - [José Javier Señaris](https://github.com/pepesenaris) | [Twitter](https://twitter.com/pepesenaris)
 - [Federico Kauffman](https://github.com/fedekau) | [Twitter](https://twitter.com/fedekauffman)
 - [Fabián Larrañaga](https://github.com/FLarra) | [Twitter](https://twitter.com/FLarraa)
-- [Pablo Marcano](https://github.com/Pablosky12)
+- [Pablo Marcano](https://github.com/Pablosky12) | [Twitter](https://twitter.com/stiv_ml)
 - [Nicolás Tinte](https://github.com/Tintef) | [Twitter](https://twitter.com/NicoTinte)
-- [Diego](https://github.com/faliure)
+- [Diego Barreiro](https://github.com/faliure)
 - [Matías Verdier](https://github.com/MatiasVerdier) | [Twitter](https://twitter.com/matiasvj)
-- [Pablo](https://github.com/pabloKz)
+- [Leonardo Fagundez](https://github.com/lfgdzdev) | [Twitter](https://twitter.com/Lfgdz)
```

Comment:
Do you know if Leonardo's surname is with or without `ú` ? I'm just asking to not write his surname wrongly because I do know some `Fagúndez`.

Reply:
You are right, the correct way to write it is `Fagúndez`. Thanks!